wrapanapi/containers/providers/openshift.py

```diff
@@ -115,3 +115,7 @@ def list_deployment_config(self):
             entity = DeploymentConfig(self, meta['name'], meta['namespace'])
             entities.append(entity)
         return entities
+
+    def get_project(self, project_name):
+        """Returns only the selected Project object"""
+        return Project(self, project_name)
```

Comment:
__QUESTION__: Why do we need this? why not just instanciate a `Project`?

Reply:
I need it for this test:
https://github.com/ManageIQ/integration_tests/pull/5933
At the moment I can only pull a list of projects with:     
def list_project(self):
I need an option to pull a specific project. 
Maybe I should just extend the list_project function like I extended the list_service and list_container_group?

Reply:
__OPTIONAL__
Just change this function name to `get_project_by_name` and I'm happy with this.